main.tf

```diff
@@ -0,0 +1,12 @@
+module "repository" {
+  for_each                  = { for repo in var.repository : repo.name => repo }
+  source                    = "./modules/repository"
+  name                      = each.value.name
+  repo_type                 = each.value.repo_type
+  supermarket_name_override = each.value.supermarket_name_override
+  description_override      = each.value.description_override
+  homepage_url_override     = each.value.homepage_url_override
+  additional_topics         = each.value.additional_topics
+  additional_status_checks  = each.value.additional_status_checks != null ? each.value.additional_status_checks : []
+  projects_enabled          = each.value.projects_enabled
+}
```

Comment:
This looks pretty good, do we need to do that null check for any of the other values?

Reply:
Nope, as the others come through properly a list cant be passed as null but a string can.